packages/react-composites/src/composites/CallComposite/selectors/mediaGallerySelector.ts

```diff
@@ -21,6 +22,6 @@ export const getRemoteParticipantsConnectedSelector = reselect.createSelector(
   [getRemoteParticipants],
   (remoteParticipants) => {
     const participants = Object.values(remoteParticipants ?? {});
-    return participants.filter((p) => p.state === 'Connected');
+    return participants.filter((p) => p.state === 'Connected' && !isMicrosoftTeamsAppIdentifier(p.identifier));
```

Comment:
Same concern here, the user should be able to see the teams voice app they are calling.